src/main/java/racingcar/ConsoleOutputHandler.java

```diff
@@ -0,0 +1,50 @@
+package racingcar;
+
+import java.util.List;
+
+public class ConsoleOutputHandler {
+    private static final RacingSnapshotProvider snapshotProvider = new RacingSnapshotProvider();
+
+    public void showStartComment() {
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+    }
+
+    public void showCommentForChoosingStageSize() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+    }
+
+    public void showSnapshotOfCurrentStage(List<Player> players) {
+
+        players.forEach(
+                player -> {
+                    System.out.println(snapshotProvider.getSnapshotOf(player));
+                }
+        );
```

Comment:
`for`를 사용하는 게 더 가독성이 좋을 때도 있습니다.
```suggestion
        for (Player player : players) {
            String snapshot = snapshotProvider.getSnapshotOf(player);
            System.out.println(snapshot);
        }
```